bin/release_tag.ts

```diff
@@ -107,7 +107,7 @@ const ask = (questionToAsk: string, callback: (answer: string) => void): void =>
 };
 
 const sendRandomGif = async (account: Account, conversationId: string, query: string): Promise<void> => {
-  const giphySearchResult = await account.service.giphy.getRandomGif(query);
+  const giphySearchResult = await account.service.giphy.getRandomGif(query, 'pg-13');
```

Comment:
What's "pg-13"? Is this documented somewhere? Please add a link to the resource where you found this.

Reply:
See [`GiphyOptions.ts#L29`](https://github.com/wireapp/wire-web-packages/blob/e905cf8c8d600091e66f3bab1e0fbb8d4fc76623/packages/api-client/src/giphy/GiphyOptions.ts#L29) and https://developers.giphy.com/docs/api/endpoint/#random.

I added it here to avoid NSFW GIFs when releasing :)

Reply:
I will add `GIPHY_RATING` to make it more clear: https://github.com/wireapp/wire-web-packages/pull/3011